app/server/lib/AttachmentFileManager.ts

```diff
@@ -106,6 +110,15 @@ export class AttachmentFileManager {
     this._docPoolId = _docInfo ? getDocPoolIdFromDocInfo(_docInfo) : null;
   }
 
+  public async shutdown(): Promise<void> {
+    if (this._loopAbort.aborted) {
+      throw new Error("shutdown already in progress");
```

Comment:
This could be fine, or it could just skip to awaiting `_transferJob`? But I expect there's a deliberate reason for throwing an error here?

Reply:
Yes, I didn't want to allow calling this method twice. The abort signal is triggered 3 lines below.